Gemfile

```diff
@@ -39,7 +39,7 @@ else
   gem 'Metrc', git: 'https://github.com/artemis-ag/Metrc.git' # rubocop:disable Bundler/DuplicatedGem
 end
 
-gem 'artemis_api', git: 'https://github.com/artemis-ag/artemis_api'
+gem 'artemis_api', git: 'https://github.com/dandehavilland/artemis_api', branch: 'feature/resource_units'
```

Comment:
Can you please create a PR to update the gem on it's repo?

Reply:
Sure, could you give me write access to that repo?

Reply:
btw, PR already exists here: https://github.com/artemis-ag/artemis_api/pull/26

Reply:
Done!